.github/workflows/build.yml

```diff
@@ -22,6 +22,21 @@ jobs:
     steps:
       - name: Check out the repo
         uses: actions/checkout@v2
+      - name: Get Short SHA
+        id: sha
+        run: echo ::set-output name=short::$(git rev-parse --short $GITHUB_SHA)
+      - name: Create Release
```

Comment:
I don't think this should be getting included - it will always trigger a release through to TEST, lets see if the tagging changes solve DEV first

Reply:
I will remove the create release